Repository/Configurations/OfficeConfiguration.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Domain.Models;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Persistence.Configurations
+{
+    class OfficeConfiguration : IEntityTypeConfiguration<Office>
+    {
+        public void Configure(EntityTypeBuilder<Office> builder)
+        {
+            builder
```

Comment:
maybe it's better to setup relation from 1 side(not many), so in MapConfiguration we will HaveOne( office) hasMany(maps) hasFK (officeId)